src/main/java/com/gtnewhorizons/gtnhgradle/modules/ModernJavaModule.java

```diff
@@ -100,22 +100,38 @@ public void apply(GTNHGradlePlugin.@NotNull GTNHExtension gtnh, @NotNull Project
         });
         ext.set("java17PatchDependenciesCfg", java17PatchDependenciesCfg);
 
+        String mixinSpec = "";
+        String lwjgl3ifySpec = "";
+
+        if (gtnh.configuration.minecraftVersion.equals("1.7.10")) {
```

Comment:
Please make the 1.7.10 and 1.12.2 strings constants somewhere to prevent typos

Reply:
I added an enum for the MC versions, I went with TauMC's Archaic/Vintage naming, but that could be changed if we wanted it to be something else. The GTNHExtension object gets the minecraft version set on creation based on what the `minecraftVersion` property is

Reply:
I'd rather this be `V1_7_10` and `V1_12_2`, archaic/vintage is too generic if we end up supporting 1.10/1.11 in the future for some reason